libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.stockPickerForm.valueChanges.subscribe(this.fetchQuote);
```

Comment:
This is not the correct way of doing the changes. If we are subscribing to valueChanges we also need to check whether the form is valid or not. Also we need to restrict the events trigger on updates of form onBlur. So I would add following on form declaration
symbol: [null, {validators: [Validators.required],updateOn: "blur"}],
period: [null, {validators: [Validators.required],updateOn: "blur"}],